entrypoint.sh

```diff
@@ -5,7 +5,7 @@ cd /usr/src/app
 ./initialize.sh &> >(tee -a /usr/src/app/log/initialisation.log)
 
 rm -f tmp/pids/server.pid
-cp /pdfcomprezzor.wasm public/pdfcomprezzor/pdfcomprezzor.wasm
+cp /pdfcomprezzor.wasm /wasm_exec.js public/pdfcomprezzor/
```

Comment:
I really do not know why this is neccessary and would consider it a bug/hack/technical debt or whatever.
It just took me ~30+ min to figure out why my dev container was serving the wrong version of `wasm_exec.js`.

Where should static content like this be saved?